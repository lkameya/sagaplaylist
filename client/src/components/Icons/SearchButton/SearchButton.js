import React from 'react';
import styles from './SearchButton.module.css';

const SearchButton = props => (
  <div className={styles.searchButton}>
    <svg width="19px" height="20px" viewBox="0 0 19 20" >
    <title>Group 4</title>
    <desc>Created with Sketch.</desc>
    <g id="Description" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
        <g id="Group-4">
            <path d="M14.7832571,10.5578395 L14.7832571,10.5578395 C15.7075431,10.5543296 16.4596703,11.3007661 16.4631802,12.2250521 C16.4632191,12.2352993 16.4631639,12.2455466 16.4630146,12.2557928 L16.3652749,18.9629733 C16.3516141,19.9004155 15.5930625,20.6560322 14.6555744,20.6660588 L14.6555744,20.6660588 C13.7445914,20.6758019 12.9981953,19.9452027 12.9884522,19.0342196 C12.9883264,19.022459 12.9883264,19.0106973 12.9884522,18.9989366 L13.060432,12.2688224 C13.0705427,11.3234782 13.8378657,10.5614295 14.7832571,10.5578395 Z" id="Rectangle" fill="#7C7B7B" transform="translate(14.725775, 15.611991) rotate(-45.000000) translate(-14.725775, -15.611991) "></path>
            <ellipse id="Oval" stroke="#7C7B7B" strokeWidth="2.75862069" cx="7.24137931" cy="7.35" rx="5.86206897" ry="5.97068966"></ellipse>
        </g>
    </g>
</svg>
  </div>

)

export default SearchButton;

