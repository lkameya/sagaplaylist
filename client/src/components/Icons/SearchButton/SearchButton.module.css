.searchButton {
  margin-left: 10px;
}