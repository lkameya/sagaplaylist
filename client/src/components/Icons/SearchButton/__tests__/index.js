import React from 'react';
import SearchButton from '../SearchButton';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

describe('Search Button', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <SearchButton>
      </SearchButton>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
