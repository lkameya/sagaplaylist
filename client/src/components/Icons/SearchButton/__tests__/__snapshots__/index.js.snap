// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Button renders correctly 1`] = `
.c0 {
  margin-left: 10px;
}

<div
  className="c0"
>
  <svg>
    search.svg
  </svg>
</div>
`;
